app/uk/gov/hmrc/plasticpackagingtax/registration/views/group/organisation_type.scala.html

```diff
@@ -57,6 +59,8 @@
     )
 }
 
+@linkHint = {<a href=@messages("organisationDetails.other.type.href") class="govuk-link" id="organisationDetails-other-type-link">@messages("organisationDetails.other.type.hint")</a>}
```

Comment:
Not a showstopper by any means but you've brought the link component in and then handcrafted one.